rpcserver.go

```diff
@@ -2657,7 +2657,7 @@ func (r *rpcServer) AssetLeafKeys(ctx context.Context,
 	// TODO(roasbeef): tell above if was tring or not, then would set
 	// below diff
 
-	leafKeys, err := r.cfg.BaseUniverse.MintingKeys(ctx, universeID)
+	leafKeys, err := r.cfg.BaseUniverse.UniverseLeafKeys(ctx, universeID)
```

Comment:
Still refs a `BaseUniverse` attribute name FWIW. IMO not a big deal though, would rather spend cycles elsewhere on more critical matters, as we can always change the names around with minimal ramifications down the line.

Reply:
Yes, this PR doesn't address all of the changes that should be made eventually.